### DISCUSSION TITLE
Формы ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#32] [Announcements]
# Формы ⚡️ Angular с примерами кода

Манипуляция формами (Angular forms) осуществляется мощными и продуманными инструментами библиотеки

https://angdev.ru/doc/angular-forms/

**Comment by khalkhalyanartur** [comment]
только знакомлюсь с Angular. В данном примере не нужно использовать 
<form #nameForm="ngForm"> ?

**Reply by khalkhalyanartur** [reply]
атрибут внутри form #nameForm="ngForm" ...
